data/entities/people.yaml

```
@@ -5622,917 +5622,2129 @@
+      value: Foresight Senior Fellow
     - label: Research Area
-      value: Existential Hope
-  tags: [existential-risk]
+      value: Quantum theory of computation
+  tags: [physics]
+  lastUpdated: 2026-03
+
+    AI safety researcher at Anthropic. PhD from UC Berkeley focused on
+    multi-agent systems and cooperative AI. Works on alignment and safety
+    evaluation of foundation models.
+  tags:
```

YAMLlint (1.38.0)

[error] duplication of key "tags" in mapping

(key-duplicates)